### PR TITLE
Fix screenshake misaligning the char on idle screenshake

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Closes https://github.com/AttorneyOnline/AO2-Client/issues/721

### CODE REVIEW

File: src/courtroom.cpp [hunk 1]
```diff
@@ -3836,7 +3836,7 @@ void Courtroom::set_self_offset(QString p_list, QString p_effect) {
       self_offset_v = 0;
     else
       self_offset_v = self_offsets[1].toInt();
-    ui_vp_player_char->move(ui_viewport->width() * self_offset / 100, ui_viewport->height() * self_offset_v / 100);
+    ui_vp_player_char->move_and_center(ui_viewport->width() * self_offset / 100, ui_viewport->height() * self_offset_v / 100);
 
     //If an effect is ignoring the users offset, we force it to the default position of the viewport.
```
[the remark this answers is not in the document]
Comment (Crystalwarrior) [reply src/courtroom.cpp:3841]: stfu

File: src/courtroom.cpp [hunk 2]
```diff
@@ -3836,7 +3836,7 @@ void Courtroom::set_self_offset(QString p_list, QString p_effect) {
       self_offset_v = 0;
     else
       self_offset_v = self_offsets[1].toInt();
-    ui_vp_player_char->move(ui_viewport->width() * self_offset / 100, ui_viewport->height() * self_offset_v / 100);
+    ui_vp_player_char->move_and_center(ui_viewport->width() * self_offset / 100, ui_viewport->height() * self_offset_v / 100);
```
Comment (in1tiate) [inline src/courtroom.cpp:3839]: this seems like it breaks offsets, have you tested this?

Comment (Crystalwarrior) [reply src/courtroom.cpp:3839]: Yeah, the offsets work with this, tested on alpha-2.10 branch actions build

**Comment by Crystalwarrior** [comment]
doesn't fully fix it, need to look into it more

**Review by oldmud0 — APPROVED** [review]
Seems like `move_and_center` should be used instead of `move` in almost all cases to correct the aspect ratio.

**Comment by stonedDiscord** [comment]
can i merge this?

**Comment by oldmud0** [comment]
yeah you're free to merge it thanks.

**Pull request merged**